htlcswitch/link_isolated_test.go

```diff
@@ -0,0 +1,253 @@
+package htlcswitch
+
+import (
+	"crypto/sha256"
+	"testing"
+	"time"
+
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+// sendHtlcBobToAlice sends an HTLC from Bob to Alice, that pays to a preimage
```

Comment:
why this move?

Reply:
It is a preparation from creating a link test context. I wanted to keep the move to a different file isolated, to make it easy to see the real changes in the next commit.